mne/viz/misc.py

```diff
@@ -1308,3 +1308,54 @@ def plot_csd(csd, info=None, mode='csd', colorbar=True, cmap=None,
 
     plt_show(show)
     return figs
+
+
+def plot_chpi_snr(snr_dict):
+    """Plot time-varying SNR estimates of the HPI coils.
+
+    Parameters
+    ----------
+    snr_dict : dict
+        The dictionary returned by `~mne.chpi.compute_chpi_snr`. Must have keys
+        ``times``, ``freqs``, ``TYPE_snr``, ``TYPE_power``, and ``TYPE_resid``
+        (where ``TYPE`` can be ``mag`` or ``grad`` or both).
```

Comment:
How about an `axes` argument that must be the same length as `snr_dict` (3.7+ dicts are always ordered) and in the same order? Internally we can (and probably should) always construct in the order `'snr', 'power', 'residual'` and note this in the docstring here

Reply:
(the alternative is to do something like an `axes` dict but that's annoying when almost all the time people will want to plot in the order snr/power/resid, and this will correspond to the `axes` in order one gets from a `fig, axes = plt.subplots(3)` call for example)

Reply:
+1 for testing `len(axes) == len(snr_dict) - 2` (because of `freqs` and `times` entries).  Also I'm fine with your proposed ordering... right now it's just alphabetical.